Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -813,5 +820,4 @@ void MainWidget::drop_event(GUI::DropEvent& event)
         open_image_fd(*result.fd, *result.chosen_file);
     }
 }
-
```

Comment:
Is this intentional? Not a big deal though.

Reply:
I guess that was clamg-format for me...
All the other blur_submenu's habe no space between them also, so I think this can stay...